bonds.md

```diff
@@ -16,3 +16,39 @@ The protocol quotes the bonder with terms for a trade at a future date. These te
 
 \*\*\*\*
 
+# Bonds
+
+Bonds allow users to buy OHM from the protocol by trading it for DAI or previously staked liquidity \(SLP Tokens\). Bonds are sold at a discount and take roughly 15 epochs to vest (ca 5 days); 
```

Comment:
One decision we may want to make is if we want to use the term SLP or LP. I would prefer LP since SLP is provider specific, in this case Sushiswap. So in the future we may have different LP and I would like to not rewrite the docs again later due to such an extension. 

```suggestion
Bonds allow users to buy OHM from the protocol for a pre-determined price, by trading it for DAI or LP Tokens. Bonds are sold at a discount and take roughly 15 epochs (5 days) to vest linearly. 
```

Reply:
> I also asked this in Discord, do we have consensus on using cash tags when referring to $OHM and $DAI? Sometimes we write $OHM and sometimes we write OHM. Should we get this streamlined?

I checked some of the Olympus medium posts, they use OHM, DAI, etc. without the cash tag. So I think we can remove the use of cash tag from now on.

Reply:
> One decision we may want to make is if we want to use the term SLP or LP. I would prefer LP since SLP is provider specific, in this case Sushiswap. So in the future we may have different LP and I would like to not rewrite the docs again later due to such an extension.

In this context, we can use LP instead of SLP as you suggested because this is an intro section. But in the next section ("How to Bond"), it makes sense to use SLP as we are referring to Sushiswap LP token specifically. So I guess we can switch between LP and SLP depending on context, what do you think?

Reply:
Makes sense.